### ISSUE TITLE
Index issue with model_vc.py

### ISSUE THREAD

**Issue by leonardw86** [freenowill/AutoVC-WavRNN#2]
I'm using the Librispeech dataset and was able to get through the first two steps of the readme file:

- try: python synthesizer_preprocess_audio.py /data to preprocess audio
- try: python synthesizer_preprocess_embeds.py /data/SV2TTS_autovc-ttsdb/synthesizer_vad -e

But when I tried the training step:

- python autovc_train.py autovc-vcc2020 /data -g

I get an index error from line 79, codes.append(torch.cat((out_forward[:,i+self.freq-1,:],out_backward[:,i,:]), dim=-1)).  When I fix the indexing error (I tried a couple of methods), I end up getting an error in the forward() routine - a dimension mismatch at line 203, c_trg_expand=c_trg.expand(-1,-1, x.size(-1)).

Anyway, can you offer any help on resolving this error?  I assumed that the preprocess_audio.py performed all the preprocessing mentioned in steps 1-9 of the readme file.
